frontend/src/app/blogs/create/page.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import React, { useState } from "react";
+import { useRouter } from "next/navigation";
+import { error } from "console";
+import { BASE_API_URL } from "@/constant";
+
+
+export default function CreateBlog() {
+    const [title, setTitle] = useState(""); // For blog title
+    const [content, setContent] = useState(""); // For blog content
+    const [link, setLink] = useState(""); // For blog link
+    const router = useRouter();
+
+    // Handle form submission
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        try {
+            // API call to create a new blog
+            const response = await fetch(BASE_API_URL + "/blogs/create", {
```

Comment:
Try using string interpolation here. It's more readable. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals